### PR TITLE
Implement command form

### PR DESCRIPTION
**Pull request by yongpingchen**
- Implement CommandForm and Builder
- Add small tests

### CODE REVIEW

File: thingif/src/main/java/com/kii/thingif/command/CommandForm.java [hunk 1]
```diff
@@ -63,21 +65,29 @@ public static Builder newBuilder() {
...
          * @return {@link Builder} instance
+         * @throws IllegalArgumentException Thrown when aliasAction is null.
          */
         @NonNull
         public Builder addAliasAction(
-                @NonNull AliasAction<? extends Action> aliasAction) {
+                @Nullable AliasAction<? extends Action> aliasAction) {
+            if (aliasAction == null) {
+                throw new IllegalArgumentException("aliasAction is null");
+            }
```
Comment (shinsuke-nara) [inline thingif/src/main/java/com/kii/thingif/command/CommandForm.java]: I think `aliasAction` should be `@NonNull` or remove `IllegalArgumentException`.

**Comment by shinsuke-nara** [comment]
@yongpingchen Could you check my comment?

File: thingif/src/main/java/com/kii/thingif/command/CommandForm.java (hunk 1 above)
Comment (yongpingchen) [inline thingif/src/main/java/com/kii/thingif/command/CommandForm.java]: ah. Should be `@NonNull`

**Comment by yongpingchen** [comment]
@shinsuke-nara  fixed. Please confirm

**Comment by shinsuke-nara** [comment]
@yongpingchen LGTM!

**Pull request merged**